.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{yml,yaml}]
+indent_size = 2
```

Comment:
For now I don't see the need to add this file to the repository.

Reply:
Sure, something to consider though as it makes it easier to maintain consistent formatting throughout